monitoring-alerting/helm/README.md

```diff
@@ -0,0 +1,18 @@
+### Installation
+
+```bash
+helm install spin stable/prometheus-operator -f values.yml -n monitoring
```

Comment:
Can we name the file `values.yaml` instead of `values.yml`?  It's a nitpick but 'all' the other helm values examples I see have the `yaml` extension and not `yml`.